spec/features/user_answers_question.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+describe "answer a question", type: :feature do
+
+  before(:each) do
+    @user_asks_question = user
```

Comment:
my preference on something like this is to eliminate the before filter altogether and create a method in this file that best describes what you are trying to accomplish.